test/factories/fis_code_location.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :fis_code_location do
+    is_important { true }
+    do_not_fetch { true }
```

Comment:
Avoid these default boolean values in this factory because they seem to cater to the specific needs of `project_test` spec. Set these flags in the spec itself, which makes the spec clearer.